sqlmesh/core/context.py

```diff
@@ -2044,6 +2043,7 @@ def test(
         verbosity: Verbosity = Verbosity.DEFAULT,
         preserve_fixtures: bool = False,
         stream: t.Optional[t.TextIO] = None,
+        log_results: bool = True,
```

Comment:
Not sure about adding this to the public context method. Can you elaborate on why this was necessary?